src/sqlancer/sqlite3/ast/SQLite3Function.java

```diff
@@ -44,6 +44,18 @@ public SQLite3Constant apply(SQLite3Constant... args) {
             }
         },
 
+        LENGTH(1, "LENGTH") {
```

Comment:
Thanks for the PR! The `apply` implementation is used only in PQS. I'm not sure if the Java string length maps directly to the SQLite one (also considering the different character encodings). Perhaps you could just return null (see https://github.com/sqlancer/sqlancer/blob/b19a41e0dcf7cbd5fd84f66694743dc12323e2ef/src/sqlancer/sqlite3/ast/SQLite3Function.java#L69), so that a `LENGTH` function is used by NoREC and TLP, but not by PQS?